### PR TITLE
Req 02 email validation create

### PR DESCRIPTION
**Pull request by morozvitali**
* Add email validation
* Add regular expression to validation

### CODE REVIEW

File: src/main/java/org/police/Main.java [hunk 1]
```diff
@@ -23,6 +54,13 @@ static void run() {
                 System.out.println("Accept denied. Please check your password");
             }
         }
-        System.out.println(accepted ? "Welcome to the magic clinic" : "Application has been blocked");
+        System.out.println(accepted ? "Welcome to the police" : "Application has been blocked");
```
Comment (oleksandr-sydorskyi) [inline src/main/java/org/police/Main.java:74]: You have a good sense of humor!:)))

**Review by oleksandr-sydorskyi — APPROVED** [review]
The implementation of email validation corresponds to the task. A good start!

File: src/main/java/org/police/Main.java [hunk 2]
```diff
@@ -31,15 +33,30 @@ static void registerNewClient() {
...
-        System.out.println("First name: ");
-        String firstName = SCANNER.nextLine();
-        client.firstName = firstName;
-        System.out.println("LastName: ");
-        String lastName = SCANNER.nextLine();
-        client.lastName = lastName;
-        return client;
+
+
+        while (true){
```
Comment (oleksandr-sydorskyi) [inline src/main/java/org/police/Main.java:38]: Looks like you missed two lines of code:
```java
        Client client = new Client();
        client.setEmail(email);
```

File: src/main/java/org/police/Main.java [hunk 3]
```diff
@@ -31,15 +33,30 @@ static void registerNewClient() {
...
+            } else {
+                System.out.println("Невірний формат імені, з великої літери без пробілів і тільки літери");
+            }
+
+
+        }
+
+
+
     }
```
Comment (oleksandr-sydorskyi) [inline src/main/java/org/police/Main.java:60]: Method buildClient() should return the Client object.
```java
return client;
```

**Review by oleksandr-sydorskyi — COMMENTED** [review]
The branch can be merged with the main one after fixing minor errors.

File: src/main/java/org/police/Main.java (hunk 2 above)
[the remark this answers is not in the document]
Comment (oleksandr-sydorskyi) [reply src/main/java/org/police/Main.java:38]: As far as I understand, Vitali has just started the project and hasn't implemented getters and setters yet.

**Comment by KaterynaMir** [comment]
Everything looks fine except for the user prompt hint in case of the invalid name: 
`System.out.println("Невірний формат імені, з великої літери без пробілів і тільки літери");`
The prompt hint written in Ukrainian could  possibly led user to the conclusion that Cyrillic input is also allowed. I would recommend to add to the prompt information that only latin letters are allowed and translate it to English.

**Pull request merged**